shared/src/question-generators/recursion/formula.ts

```diff
@@ -17,27 +17,29 @@ const translations: Translations = {
     basecase: "The base case is",
     text1: "Consider the following recursive procedure `{{0}}` with integer input ${{1}}$:",
     text2: "Let ${{0}}$ be the number of stars (`*`) that the procedure prints.",
-    "long-name": "Recurrence Relation",
+    longName: "Recurrence Relation",
     question: "What is the recurrence relation of",
     name: "Recurrence",
     description: "Determine the recurrence relation of a recursive function",
     bottomnote: "Note: This field expects a string of the form `{{0}}` as input.",
-    "feedback.incomplete": "Incomplete or too complex",
+    feedbackIncomplete: "Incomplete or too complex",
+    invalidName: "Invalid function name, please use: {{0}} and {{1}}",
   },
   de: {
     basecase: "Der Basisfall ist",
     text1: "Betrachte die folgende rekursive Prozedur `{{0}}` mit ganzzahliger Eingabe ${{1}}$:",
     text2: "Sei ${{0}}$ die Anzahl der Sterne (`*`), die die Prozedur ausgibt.",
-    "long-name": "Rekurrenzrelation",
+    longName: "Rekurrenzrelation",
     question: "Was ist die Rekurrenzrelation von",
     name: "Rekurrenz",
     description: "Bestimme die Rekurrenzrelation einer rekursiven Funktion",
     bottomnote: "Hinweis: Dieses Feld erwartet einen String der Form `{{0}}` als Eingabe.",
-    "feedback.incomplete": "Nicht vollständig oder zu komplex",
+    feedbackIncomplete: "Nicht vollständig oder zu komplex",
+    invalidName: "Ungültiger Funktionsname, bitte verwende: {{0}} und {{1}}",
   },
 }
 
-export const RecursionFormula: QuestionGenerator = {
+export const recursionFormula: QuestionGenerator = {
```

Comment:
The generators are class-like, even though they are not actually classes. Currently all generator objects are named in PascalCase. So this should stay consistent. If there is an issue with the linter?

Reply:
I will change it back, but maybe we should change the linter, because I get `...must match one of the following formats: camelCase, UPPER_CASE...` error.

Reply:
I see, then we should change the linter and just add PascalCase.